### ISSUE TITLE
Is it possible to build a top navigation without permalinks ?

### ISSUE THREAD

**Issue by DataSaiyentist** [chrisrhymes/bulma-clean-theme#145]

**Comment by magoerlich** [comment]
## TL;DR and nice answer:

Go here to get help https://jekyllrb.com/docs/community/

## More exhaustive, but (a bit) mean answer:
I wouldn't say you're bad, I'd rather say you're missing a fundamental understanding of both: the tools you try to use and what you try to achieve with them.

You can do whatever you want and nothing of this is an issue related to this theme, but your understanding of how jekyll works.
So try to do the bare minimum and read the docs, work through the basic tutorial and learn about permalinks, includes and layouts. 
https://jekyllrb.com/docs/

If you need help with that, ask the jekyll community and don't randomly open issues on repos unrelated to your issue:
https://talk.jekyllrb.com/

The theme doesn't care how you **format** your permalinks and a jekyll website without any permalinks would be quite useless unless it's a single-page-website.
If you had read **at least** about the [basic of how jekyll works](https://jekyllrb.com/docs/rendering-process/), you'd know that your *.md files get converted to *.html

So what you want is propbably:

```
- name: Some articles
  dropdown:
    - name: Article 1
      link: /article1/index.html
    - name: Article 2
      link: /article2/index.html
```
(.html, not .md)

And for the PDF File, put it in the /assets directory and link to it like that:
```
- name: Page2
  link: /assets/file.pdf
```
https://jekyllrb.com/docs/posts/#including-images-and-resources

Wasting other peoples time by not doing your homework is disrespectful.
